RecoLocalCalo/CaloTowersCreator/src/CaloTowersCreator.cc

```diff
@@ -62,7 +63,8 @@ CaloTowersCreator::CaloTowersCreator(const edm::ParameterSet& conf) :
         conf.getParameter<double>("MomHBDepth"),
         conf.getParameter<double>("MomHEDepth"),
         conf.getParameter<double>("MomEBDepth"),
-        conf.getParameter<double>("MomEEDepth")
+        conf.getParameter<double>("MomEEDepth"),
+        conf.exists("HcalPhase") ? conf.getParameter<int>("HcalPhase") : 0
```

Comment:
@slava77 - I posted a comment about this 2 weeks ago when we originally discussed it: https://github.com/cms-sw/cmssw/pull/11497#issuecomment-143335200

It is my understanding that fillDescriptions will not be sufficient here, because different HLT configs use different settings for calotowermaker. If there is some way around this, please explain.